### PR TITLE
studio: polish asset grid

### PR DESCRIPTION
**Pull request by achen5671**
## Summary
https://tsu.atlassian.net/browse/IR-3389

## References
closes #_insert number here_

### CODE REVIEW

**Review by SYBIOTE — APPROVED** [review]

File: packages/ui/src/components/editor/panels/Files/icon/index.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ export const FileIcon = ({
           alt=""
         />
       ) : FallbackIcon ? (
-        <FallbackIcon className={`${color}`} />
+        <FallbackIcon className={`${color} h-full w-full`} />
```
Comment (aditya-mitra) [inline packages/ui/src/components/editor/panels/Files/icon/index.tsx]: make the fallback occupy 85-90% instead of full
image

**Review by aditya-mitra — CHANGES_REQUESTED** [review]
Can you also fix the [toolbar in assets panel](https://www.figma.com/design/DXwFHGa5g0peqqSjyjpxQz/Studio-Design?node-id=5269-19471&t=s1gIxJxZoZvc9zg5-0)?

[image
](https://www.figma.com/design/DXwFHGa5g0peqqSjyjpxQz/Studio-Design?node-id=5269-19471&t=s1gIxJxZoZvc9zg5-0)

**Comment by achen5671** [comment]
> Can you also fix the [toolbar in assets panel](https://www.figma.com/design/DXwFHGa5g0peqqSjyjpxQz/Studio-Design?node-id=5269-19471&t=s1gIxJxZoZvc9zg5-0)?
> 
> [image ](https://www.figma.com/design/DXwFHGa5g0peqqSjyjpxQz/Studio-Design?node-id=5269-19471&t=s1gIxJxZoZvc9zg5-0)

yes, I think it'll be better if I do this in a separate PR

File: packages/ui/src/components/editor/panels/Files/icon/index.tsx (hunk 1 above)
Comment (achen5671) [reply packages/ui/src/components/editor/panels/Files/icon/index.tsx]: I added some padding around

**Review by aditya-mitra — APPROVED** [review]

**Pull request merged**